elements/rh-avatar/rh-avatar.ts

```diff
@@ -10,10 +10,14 @@ import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/c
-  saturated: '',
+const DEFAULT_AVATARS = {
+  light: html`<svg xmlns="http://www.w3.org/2000/svg" style="enable-background:new 0 0 36 36" viewBox="0 0 36 36" role="presentation" part="img">
```

Comment:
we could perhaps achieve further gains by using the `svg` template tag and interpolating the dark or light colours, or (maybe even better) just by using css on the inline svg

Reply:
We could. And I had considered it. I've had some issues in some of my own work in the past with colors not updating, but I can't remember if it was resolved by a requestUpdate() or if it was how a particular browser cached the SVG.